cmd/mdatagen/go.mod

```diff
@@ -52,7 +52,7 @@ require (
 	go.opentelemetry.io/collector/pdata/pprofile v0.115.0 // indirect
 	go.opentelemetry.io/collector/pdata/testdata v0.115.0 // indirect
 	go.opentelemetry.io/collector/pipeline v0.115.0 // indirect
-	go.opentelemetry.io/collector/processor/processorprofiles v0.115.0 // indirect
+	go.opentelemetry.io/collector/processor/xprocessor v0.115.0 // indirect
```

Comment:
this will break contrib releases, see https://github.com/open-telemetry/opentelemetry-collector/pull/11884, please use commit version everywhere

Reply:
I cannot use commit version because commit will be changed once it's merged

Reply:
Fair enough - `xprocessor` version can be fixed in a follow-up PR